docs/store/artifacts.md

```diff
@@ -56,15 +61,15 @@ The default `artifact_path` can be specified at the cluster level, client level,
 You can set the default `artifact_path` for your environment using the {py:func}`~mlrun.set_environment` function.
 
 You can override the default `artifact_path` configuration by setting the `artifact_path` parameter of 
-the {py:func}`~mlrun.set_environment` function. You can use variables in the artifacts path, 
+the {py:func}`~mlrun.projects.MlrunProject.artifact_path` function. You can use variables in the artifacts path, 
```

Comment:
This is not accurate - by setting the `MlrunProject.artifact_path` property you override the default `artifact_path` configuration **for that specific project**.